Themes/_fallback/Languages/sk.ini

```diff
@@ -0,0 +1,3049 @@
+SmoothLines=Zapína/vypína zobrazenie radaru Groove s použitím vyhladených čiar.
+Center1Player=Zapnite tuto voľbu, ak chcete zobrazovať kroky jednoho hráča v strede obrazovky.
+SongsPerPlay=Počet skladieb, ktoré si môže hráč zahrať v normálnom režime (pri vypnutom režime Eventu)
+SoundEffect=
+SoundResampleQuality=Pokročilé: Vyberte kvalitu prevzorkovania.
+SoundVolume=Upravte úroveň hlasitosti počas hry.
+SoundVolumeAttract=Upravte úroveň hlasitosti v režime demo.
+Speed=Rýchlosť
+Speed Increment=Veľkosť menších krokov použitých na úpravu rýchlosti.
+Speed Increment Large=Veľkosť väčších krokov použitých na úpravu rýchlosti.
+StepMania Credits=Vývojársky tím StepManie.
+Steps=Kroky
+StretchBackgrounds=Zapína/vypína rozťahovanie pozadí.
+Style=
+Test Input=Vyskúšajte citlivosť pripojených joystickov.
+Test Lights=Vyskúšajte citlivosť pripojených svetiel.
+TextureColorDepth=Vyberte farebnú hĺbku textúr. 32bitové textúry spotrebujú viac pamäte, ale vyzerajú krajšie.
+Theme=Vyberte zo zoznamu nainštalovaných balíčkov motívov.
+ThreeKeyNavigation=&oq;Päť kláves&oq;: tlačidlá ponuky Ľ/P/D/H a Start.  &oq;Tri klávesy&oq;: tlačidlá ponuky Ľ/P a Start.
+TimingWindowScale=Zvýšením tejto hodnoty budú Vaše kroky hodnotene prísnejšie (t. j. s nižšou toleranciou načasovania).
+Toggle Song=Prepnúť skladbu
+Turn=Otočenie
+Transfer Edits to USB=Preniesť úpravy na USB
+Transfer Edits from USB=Preniesť úpravy z USB
+UseUnlockSystem=Povoliť alebo zakázať systém odomykania skladieb.
+VisualDelaySeconds=Kalibrovať oneskorenie zobrazenia.
+Vsync=Prispôsobiť frekvenciu snímok obnovovacej frekvencii Vášho monitora.  Táto voľba je aktívna pri v režime zobrazenia na &oq;celú obrazovku&cq;.
+WideScreen16_9=Zapnúť zobrazenie na širokouhlú obrazovku.
+WideScreen16_10=Zapnúť zobrazenie na širokouhlú obrazovku.
+Windowed=Zvoľte, či chcete aby sa hra zobrazovala na celej obrazovke alebo v okne.
+
+#
+UI Options=Zmeňte používateľské rozhranie platné v celej hre.
+Input Options=Zmeňte nastavenia vstupu ako je oneskorenie a rýchlosť.
+Arcade Options=Zmeňte vlastnosti arkády ako je napr. časovač ponúk.
+Select Game=Zvoľte si váš obľúbený typ hry.
+System Direction=Upravte najčastejšie menené nastavenia.
+
+#
+OnlyPreferredDifficulties=Iba vybrané obtiažnosti.
+#
+PercentageScoring=Vyberte medzi zobrazením podrobného hodnotenia alebo percent.
+
+# course
+Select Course=Upraviť tento maratón.
+Max Minutes=Ako dlho má najviac trvať maratón.
+Choose Song=Vyberte skladbu pre toto miesto v maratóne.
+Choose Steps=Vyberte typ krokov pre túto skladbu.
+
+Play=Spustiť maratón.
+Save=Uložiť maratón.
+Edit Course=Upraviť obsah maratónu.
+Shuffle=Premiešať skladby.
+#
+GamePrefDefaultFail=Okamžitý fail spôsobí, že po vyčerpaní zdravia hráč okamžite skončí. Immediate fail causes a player to die when their life bar reaches 0. Oneskorený umožní hráčovi v hre pokračovať.
```

Comment:
> Immediate fail causes a player to die when their life bar reaches 0.

Is this supposed to be english yet?

Reply:
> > Immediate fail causes a player to die when their life bar reaches 0.
> 
> Is this supposed to be english yet?

Thank you, good catch. I am still working on the translation, please put this on hold for now. Should have an update in a few weeks.

Reply:
No problem mate! Was just making sure if it was intentional or not

Reply:
> No problem mate! Was just making sure if it was intentional or not

Thanks for checking. I am now done with the changes - tested in-game and it looks pretty decent. Could you please review again?